entc/gen/template/dialect/sql/query.tmpl

```diff
@@ -249,11 +249,19 @@ func ({{ $receiver }} *{{ $builder }}) sqlCount(ctx context.Context) (int, error
 }
 
 func ({{ $receiver }} *{{ $builder }}) sqlExist(ctx context.Context) (bool, error) {
-	n, err := {{ $receiver }}.sqlCount(ctx)
-	if err != nil {
+	{{- if $.HasOneFieldID }}
+	_, err := {{ $receiver }}.FirstID(ctx)
+	{{- else }}
+	_, err := {{ $receiver }}.First(ctx)
+	{{- end }}
+	switch {
```

Comment:
```suggestion
	switch _, err := {{ $receiver }}.First{{ if $.HasOneFieldID }}ID{{ end }}(ctx); {
```

Reply:
Not sure if you see my suggestion correctly because I see this:
<img width="849" alt="image" src="https://user-images.githubusercontent.com/7413593/188281003-76c768e5-df85-4e21-877e-629cc2cef9b8.png">

But I mean this:
<img width="682" alt="image" src="https://user-images.githubusercontent.com/7413593/188281020-a3304e52-1b59-47ea-a17d-3ec3fd9bb8e6.png">